frontend/front/src/pages/Surveyor/account/edit/EditAccount.js

```diff
@@ -38,168 +61,195 @@ const EditAccount = () => {
     lineHeight: "1.25rem",
   };
 
+  const handleUpdateAccount = useCallback(
+    async (values) => {
+      const updatedAccount = await updateAccount({
+        id,
+        body: {
+          surveyor: {
+            firstname: values.firstName,
+            lastname: values.lastName,
+            phone: values.phoneNumber,
+            email,
+            street_address: values.address,
+          },
+        },
+      });
+      return updatedAccount;
```

Comment:
You should make sure this updates the `user` object as well, otherwise if they change their email here they won't be able to use that to log in.

Reply:
Meeting note: Maybe surveyor shouldn't have email at all, just user.

Reply:
@mattdelsordo I'm not entirely sure how to update a user.

Reply:
I looked into this a bit, and it seems like this might be a bit more complicated than just doing a PATCH. I say for now, just don't let the user edit their email or password.